SECURITY.md

```diff
@@ -0,0 +1,122 @@
+
+The VRT lead should notify the reporter of the classification and the reasoning behind the VRT’s decision within 7 days of the acknowledgement email. This notification should include links to any issues that were opened as a result of the report. For problems classified as bugs, feature requests, or low-severity vulnerabilities, issues should be opened and assigned to the relevant team.
+
+The rest of this document describes how we handle high-severity vulnerabilities.
+
+## Remediate
+
+### Create a Coordination Channel
+
+If the problem is confirmed as a high-severity vulnerability, the VRT will open a [GitHub security advisory](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories) and, if both the VRT and reporter agree, add the reporter to the advisory so we can collaborate on it. We will coordinate work on the vulnerability on:
+
+* **The GitHub security advisory.** These advisories will let us collaborate in private, and they are appropriate in cases where disclosing the vulnerability prior to remediating it could put our users or developers at risk.
+* **(Optionally) An additional GitHub issue and pull request.** This will immediately disclose the vulnerability, and we will take this approach when immediate disclosure poses little risk to our users and developers. For example, when the vulnerability is already publicly known. Unlike security advisories, CI checks can run on these PRs.
+
+Note that we should create a GitHub security advisory even if we choose to collaborate on a fix using an open issue or PR because we’ll want to publish the advisory when disclosing the vulnerability.
+
+### Request a CVE
+
+A CVE should be requested for all security vulnerabilities. Since we create a GitHub Security Advisory, we can [get a CVE from GitHub](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories#cve-identification-numbers). As a backup, we can [get a CVE from MITRE](https://cveform.mitre.org/).
+
+### Develop a Patch
+
+Regardless of which approach we take, the VRT will prioritize developing a patch to fix the vulnerability. If the vulnerability is easily exploitable, remediation should take priority over all other work. Remediation should be completed within 7 days of the report being classified as a high-severity vulnerability.
+
+## Disclose
+
+We generally follow a 90-day disclosure timeframe, meaning that we ask that reporters give us 90 days to fix vulnerabilities before publicly disclosing them. 90 days should be viewed as an upper bound, and we should aim to remediate vulnerabilities as quickly as possible. In all cases, the disclosure timeline should be explicitly coordinated with the reporter, and we should arrange to publicly disclose the vulnerability simultaneously with the reporter. Our disclosure will include credit for the reporter if they so wish.
```

Comment:
This paragraph uses 'should' a lot--is this much hedging necessary here?

Ditto elsewhere in the doc--we use 'should' quite a bit.

Reply:
I removed some of the "should"s